boot/zephyr/Kconfig

```diff
@@ -493,6 +493,18 @@ config BOOT_SERIAL_CDC_ACM
 
 endchoice
 
+config MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD
+	bool "Allow to select image number for DFU"
+	help
+	  With the option enabled, the mcuboot serial recovery will
+	  respect the "image" field in mcumgr image update frame
+	  header.
+	  The mapping of image number to partition is as follows:
+	    0 -> image-0 (slot 0)
```

Comment:
I would use primary/secondary terminology as well:
0 -> image-0 (primary slot of the firs images pair)
1 -> image-0 (secondary slot of the firs images pair)

Reply:
OK, will change.